L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCMulticlustersSA.cc

```diff
@@ -0,0 +1,191 @@
+#include "DataFormats/L1THGCal/interface/HGCalMulticluster.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
+#include "L1Trigger/L1THGCalUtilities/interface/HGCalTriggerNtupleBase.h"
+
+class HGCalTriggerNtupleHGCMulticlustersSA : public HGCalTriggerNtupleBase {
```

Comment:
Instead of duplicating the multi-cluster ntuple, might be better to add variables in the existing one, and eventually have a flag specifying which sets of variables we want to store

Reply:
I've modified the existing multicluster ntuple to include the additional hw based variables, and a python config option specifying whether to store these or not.